src/CodeBlock.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SilverStripe\MD_PHP_CodeSniffer;
```

Comment:
```suggestion
namespace SilverStripe\MarkdownPhpCodeSniffer;
```
Bit of a non-standard namespace - maybe globally change it to this? We don't need it to match PHP_CodeSniffer style

Reply:
This hasn't been implemented?

Reply:
This hasn't been implemented?

Reply:
Oops, somehow missed that one.
Done now.